catalog/task/run-file-updates/0.2/run-file-updates.yaml

```diff
@@ -0,0 +1,65 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: run-file-updates
+  labels:
+    app.kubernetes.io/version: "0.2"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to run file updates
+  params:
+    - name: jsonKey
+      type: string
+      description: The json key containing the file updates
+      default: ".spec.extraData.fileUpdates[]"
+    - name: fileUpdatesPath
+      type: string
+      description: The path to the file containing the file updates
+    - name: snapshotPath
+      type: string
+      description: Path to the JSON string of the Snapshot spec in the config workspace to apply the mapping to
```

Comment:
The workspace is called "data" in this case.

Reply:
fixed. I blindly copied this one.